replay-handler-cloudformation.yaml

```
@@ -48,7 +48,7 @@ Parameters:
     Description: Number of EC2 instances in the replay handler.
   KeyName:
     Description: This parameter is optional. The name of the EC2 Key Pair to allow SSH access to the instances.
-    Type: 'String'
+    Type: 'AWS::EC2::KeyPair::KeyName'
```

`Type: 'String'` was choosen because it allows empty value (no key pair, meaning key is optional). `Type: 'AWS::EC2::KeyPair::KeyName'` does not allow empty value.

```suggestion
    Type: 'String'
```